tests/test_opSimOrdering.py

```diff
@@ -0,0 +1,76 @@
+import os
+import numpy as np
+import pandas as pd
+from desc.twinkles import OpSimOrdering
+from lsst.utils import getPackageDir
+import unittest
+
+class TestOpSimOrdering(unittest.TestCase):
+
+    def setUp(self):
+        self.opSimDBPath = '/Users/rbiswas/data/LSST/OpSimData/minion_1016_sqlite.db'
```

Comment:
Maybe we should have this be accessed from an environment variable.   I would really like to get this package running regularly in Jenkins at SLAC, but I need someone with admin privileges to set up the web hook as I described [here](https://github.com/DarkEnergyScienceCollaboration/Twinkles/issues/72#issuecomment-253580705) (@drphilmarshall Could you find a moment to do that please?).